src/main/java/use_case/login/LoginDataAccessInterface.java

```diff
@@ -0,0 +1,11 @@
+package use_case.login;
+
+import entity.User;
+public interface LoginDataAccessInterface {
+
+    boolean existsByName(String identifier);
```

Comment:
I thought we agreed to always use the word "username" instead of "name" throughout the code. It doesn't really matter functionally, but it's just to avoid any sort of confusion that this may cause for other readers of the code.